lib/backend/etcdbk/etcd.go

```diff
@@ -335,12 +338,13 @@ func (b *EtcdBackend) reconnect(ctx context.Context) error {
 	tlsConfig.ClientCAs = certPool
 
 	clt, err := clientv3.New(clientv3.Config{
-		Endpoints:   b.nodes,
-		TLS:         tlsConfig,
-		DialTimeout: b.cfg.DialTimeout,
-		DialOptions: []grpc.DialOption{grpc.WithBlock()},
-		Username:    b.cfg.Username,
-		Password:    b.cfg.Password,
+		Endpoints:          b.nodes,
+		TLS:                tlsConfig,
+		DialTimeout:        b.cfg.DialTimeout,
+		DialOptions:        []grpc.DialOption{grpc.WithBlock()},
+		Username:           b.cfg.Username,
+		Password:           b.cfg.Password,
+		MaxCallSendMsgSize: b.cfg.MaxClientMsgSizeBytes,
```

Comment:
What is the default value gonna be if it's not set in the config? Can it just be set to 10MB or something?

Reply:
It will match the etcd's client default if unset (which it is by default). I think it's a better strategy since if I used a specific value which is a default in etcd version X it might get changed in etcd version X+1 and will no longer match the default while zero value will likely continue to be treated as unset.